### DISCUSSION TITLE
NetworkInformation

### DISCUSSION THREAD

**Discussion by wkmhv** [unoplatform/uno#17960] [Q&A] [answered]
![grafik](https://github.com/user-attachments/assets/73271dcf-3aa7-4c38-94cd-0c27b6503e02)

https://platform.uno/docs/articles/features/windows-networking.html

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @wkmhv,

If I'm not mistaken, you need to observe it directly from NetworkInformation, like this:

```csharp
NetworkInformation.NetworkStatusChanged += NetworkInformation_NetworkStatusChanged;

private void NetworkInformation_NetworkStatusChanged(object sender)
{
    // Your implementation here
}
```
This implementation should work. There might be an issue in our documentation—I’ll investigate further. Thanks for bringing this to my attention.

**Reply by wkmhv** [reply]
Work's. Thank you
